caramel/models.py

```diff
@@ -135,10 +132,10 @@ class CSR(Base):
     orgunit = _sa.Column(_sa.String(_UB_OU_LEN))
     commonname = _sa.Column(_sa.String(_UB_CN_LEN))
     rejected = _sa.Column(_sa.Boolean(create_constraint=True))
-    accessed = _orm.relationship(
+    accessed: _orm.RelationshipProperty = _orm.relationship(
```

Comment:
This doesn't really match the type expected, it should be:

from typing import List


    accessed: List[AccessLog]  = _orm.relationship(
        "AccessLog", backref="csr", order_by="AccessLog.when.desc()"
    )
    certificates: List[Certificate] = _orm.relationship(
        "Certificate",
        backref="csr",
        order_by="Certificate.not_after.desc()",
        lazy="dynamic",
        cascade="all, delete-orphan",
    )

Reply:
The problem here is that Certificate and AccessLog is used before assignment, which pylint makes a fuss about, and changing the order to CSR-class after Certificate and AccessLog we have the same problem, because AccessLog uses CSR. One option is to say they are lists of Base but that sounded to prone for confusion("Base" is very generic as a name) so thats why it ended up like that. I guess I could suppress the error, but that feels naughty.

Reply:
Then the fix is to do:

`List["Accesslog"]`    Quotes around the class name makes it a safe forward-declaration.

Reply:
Didn't know that, thanks! will do